dotCMS/src/main/java/com/dotcms/publisher/ajax/RemotePublishAjaxAction.java

```diff
@@ -557,6 +555,7 @@ public void downloadUnpushedBundle ( HttpServletRequest request, HttpServletResp
         final Map<String, String> map = getURIParams();
         final String bundleId = map.get( "bundleId" );
         final String paramOperation = map.get( "operation" );
+        final String bundleFilter = UtilMethods.isSet(map.get("filterKey")) ? map.get("filterKey") : "";
```

Comment:
Codacy found an issue: [Avoid declaring a variable if it is unreferenced before a possible exit point.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5896739)